Resources/Locale/en-US/botany/components/plant-analyzer-component.ftl

```diff
@@ -0,0 +1,105 @@
+
+plant-analyzer-component-environemt = This [color=green]{$seedName}[/color] requires an atmosphere at a pressure level of [color=lightblue]{$kpa}kPa ± {$kpaTolerance}kPa[/color], temperature of [color=lightsalmon]{$temp}°k ± {$tempTolerance}°k[/color] and a light level of [color=white]{$lightLevel} ± {$lightTolerance}[/color].
+plant-analyzer-component-environemt-void = This [color=green]{$seedName}[/color] has to be grown [bolditalic]in the vacuum of space[/bolditalic] at a light level of [color=white]{$lightLevel} ± {$lightTolerance}[/color].
+plant-analyzer-component-environemt-gas = This [color=green]{$seedName}[/color] requires an atmosphere containing [bold]{$gases}[/bold] at a pressure level of [color=lightblue]{$kpa}kPa ± {$kpaTolerance}kPa[/color], temperature of [color=lightsalmon]{$temp}°k ± {$tempTolerance}°k[/color] and a light level of [color=white]{$lightLevel} ± {$lightTolerance}[/color].
+
+plant-analyzer-produce-plural = {MAKEPLURAL($thing)}
+plant-analyzer-output = {$yield ->
+    [0]{$gasCount ->
+        [0]The only thing it seems to do is consume water and nutrients.
+        *[other]The only thing it seems to do is turn water and nutrients into [bold]{$gases}[/bold].
+    }
+    *[other]It has [color=lightgreen]{$yield} {$potency}[/color]{$seedless ->
+        [true]{" "}but [color=red]seedless[/color]
+        *[false]{$nothing}
+    }{" "}{$yield ->
+        [one]flower
+        *[other]flowers
+    }{" "}that{$gasCount ->
+        [0]{$nothing}
+        *[other]{$yield ->
+            [one]{" "}emits
+            *[other]{" "}emit
+        }{" "}[bold]{$gases}[/bold] and
+    }{" "}will turn into{$yield ->
+        [one]{" "}{INDEFINITE($firstProduce)} [color=#a4885c]{$produce}[/color]
+        *[other]{" "}[color=#a4885c]{$producePlural}[/color]
+    }.{$chemCount ->
+        [0]{$nothing}
+        *[other]{" "}There are trace amounts of [color=white]{$chemicals}[/color] in its stem.
+    }
+}
+
+plant-analyzer-potency-tiny = tiny
+plant-analyzer-potency-small = small
+plant-analyzer-potency-below-average = below-average sized
+plant-analyzer-potency-average = average sized
+plant-analyzer-potency-above-average = above-average sized
+plant-analyzer-potency-large = rather large
+plant-analyzer-potency-huge = huge
+plant-analyzer-potency-gigantic = gigantic
+plant-analyzer-potency-ludicrous = ludicrously large
+plant-analyzer-potency-immeasurable = immeasurably large
+
+plant-analyzer-print = Print
+plant-analyzer-printout-missing = N/A
+plant-analyzer-printout = [color=#9FED58][head=2]Plant Analyzer Report[/head][/color]{$nl
+    }──────────────────────────────{$nl
+    }[bullet/] Species: {$seedName}{$nl
+    }{$indent}[bullet/] Viable: {$viable ->
+        [no][color=red]No[/color]
+        [yes][color=green]Yes[/color]
+        *[other]{LOC("plant-analyzer-printout-missing")}
+    }{$nl
+    }{$indent}[bullet/] Endurance: {$endurance}{$nl
```

Comment:
likewise for indentation, you can just indent it further than established at the start of the block and it'll work as expected

Reply:
Same as for the newline.